src/authenticated/member/MemberProfileEditPage.js

```diff
@@ -32,15 +34,23 @@ const updateUserQuery = gql`
         id
         name
       }
+      projectsChampioned {
+        id
+        name
+      }
     }
   }
 `;
 
 function prepUserForUpdate(updatedUser) {
   const skillsIds = updatedUser.skills.map(skill => skill.id);
+  const projectsChampionedIds = updatedUser.projectsChampioned.map(
+    project => project.id
+  );
   return {
     ...updatedUser,
-    skillsIds
+    skillsIds,
+    projectsChampionedIds
   };
 }
 
```

Comment:
We still want `refetchQueries: ['getProject']` on the `updateUserQuery` grqphql query. I think that is the last thing to update and then we can get this merged

Reply:
Ok, I thought I added that before but I guess not.  This PR was a little messy. It should be ready to merge now.

Reply:
Hmm, the project doesn't seem to be updating...